eventespresso/event-espresso-core
# Improve Event Manager Role Check in Ticket Selector

This PR:

- extracts Event Manager logic from `core/domain/services/admin/events/editor/EventManagers` into separate class that should be more usable by others

- extracts Current User just added to `DisplayTicketSelector` into new class for same reasons as above BUT uses the new EventManagers class to determine if user is an Event Manager instead of simply checking for 'Administrator' role.

- updates references to the above logic and does some extra formatting and stuff
### testing


- LOG OUT

- log in again as an "Event Administrator" User

- visit registration page on frontend

- [ ] confirm that "Public", "Members Only", and "Admins Only" tickets are visible

- go to event lists admin page and click on the "Registrations" link under the name of the event from above

- click "Add New Registration" button at top of screen

- [ ] confirm that "Public", "Members Only", "Admins Only", and "Admin UI Only" tickets are visible
